CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Since v0.0.10, this Changelog is formatted according to the [Common Changelog][c
 
 ### Changed
 - ([#1321](https://github.com/demos-europe/demosplan-ui/pull/1321)) BREAKING: Remove DpFormRow ([@salisdemos](https://github.com/salisdemos))
+- ([#1323](https://github.com/demos-europe/demosplan-ui/pull/1323)) DpAccordion: Remove default submit behaviour and adjust styles ([@rafelddemos](https://github.com/rafelddemos))
```

Comment:
I think this maybe happened during a merge, but there is already a ### Changed block under ## Unreleased, and we should move these two entries to that block (below)